components/atoms/TextInput/text-input.tsx

```diff
@@ -61,9 +61,12 @@ const TextInput = ({
             name={name}
             id={id || name || ""}
             placeholder={placeholder || ""}
-            className={`flex-1 focus:outline-none  ${
-              disabled && "bg-light-slate-3  cursor-not-allowed  text-light-slate-9"
-            } `}
+            className={clsx(
+              "flex-1 focus:outline-none",
+              disabled && "bg-light-slate-3 cursor-not-allowed text-light-slate-9",
+              classNames
+            )}
+
```

Comment:
```suggestion
```

Reply:
@NsdHSO Is there a change I need to make?

Reply:
Yes, could you please remove this empty line ??